style.css

```diff
@@ -91,6 +90,283 @@
+  width: 325px;
+  height: 300px;
+  margin: 0 auto;
+  background-image: url("images/other-projects.png");
+  display: flex;
+  align-items: flex-end;
+}
+
+.other-project-title {
+  font-family: "Crete Round", serif;
+  font-weight: 400;
+  font-size: 32px;
+  color: white;
+  margin: 0 0 20px 20px;
+}
+
+.other-project-description {
+  font-family: "Inter", sans-serif;
+  color: white;
+  margin: 0 0 20px 20px;
+}
+
+.other-project-tags {
+  display: flex;
+  list-style-type: none;
+}
+
+.other-project-tag {
+  font-family: "Inter", sans-serif;
+  color: white;
+  margin: 0 10px 20px 20px;
+  font-size: 15px;
+  background-color: #5c5c69;
+  padding: 10px;
+}
+
+.other-project-see-project-btn {
+  background-color: #ff6b00;
+  color: white;
+  font-family: "Inter", sans-serif;
+  font-size: 14px;
+  font-weight: bold;
+  text-decoration: none;
+  border: none;
+  cursor: pointer;
+  width: 100%;
+  display: block;
+  text-align: center;
+  padding: 10px 0;
+}
+
+.other-project-see-project-btn:hover {
+  background-color: #fff5e1;
+  color: #ff6b00;
+}
+
+.other-project-see-project-btn:focus {
+  background-color: #ff6b00;
+  color: white;
+}
+
+.other-project-see-project-btn:disabled {
+  background-color: #f4f1f0;
+  color: #5b5957;
+}
+
+.about-me-section {
+  margin-top: 100px;
+  width: 100%;
+  background-color: #1c1a19;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  position: relative;
+  padding: 50px 0;
+  overflow: hidden;
+  background-position: right bottom;
+  background-image: url(images/illustration-about-me-mobile.png);
+  background-repeat: no-repeat;
+}
+
+.about-me-title {
+  font-family: "Crete Round", serif;
+  font-weight: 400;
+  font-size: 72px;
+  color: #ff6b00;
+  margin: 0 0 20px 20px;
+}
+
+.about-me-description {
+  color: white;
+  font-family: "inter", sans-serif;
+  font-size: 18px;
+  padding: 0 20px;
+  margin-right: 60px;
+}
+
+.get-my-resume-btn {
+  background-color: #ff6b00;
+  color: white;
+  font-family: "Inter", sans-serif;
+  font-size: 14px;
+  font-weight: bold;
+  text-decoration: none;
+  border: none;
+  cursor: pointer;
+  margin: 20px 20px;
+  display: block;
+  width: 130px;
+  height: 40px;
+  text-align: center;
+  padding: 10px 0;
+}
+
+.get-my-resume-btn:hover {
+  background-color: #fff5e1;
+  color: #ff6b00;
+}
+
+.get-my-resume-btn:focus {
+  background-color: #ff6b00;
+  color: white;
+}
+
+.get-my-resume-btn:disabled {
+  background-color: #f4f1f0;
+  color: #5b5957;
+}
+
+.about-me-illustration {
+  position: absolute;
+  right: -200;
+  bottom: 0;
+  width: 300px;
+}
+
+.skills-section {
+  width: 100%;
+  background-color: #1c1a19;
+  padding-top: 50px;
+  padding-bottom: 50px;
+}
+
+.about-me-line {
+  width: 80%;
+  height: 1px;
+  margin: 0 auto;
+  background-color: #5c5c69;
+}
+
+.skills-wrapper {
+  display: grid;
+  grid-template-columns: 1fr;
+  font-family: "inter", sans-serif;
+  font-size: 32px;
+  color: white;
+  list-style-type: none;
+}
```

Comment:
- [ ] You've done an excellent job organizing the skills section of your portfolio :+1:. However, it would be beneficial if you could adjust the spacing between the elements to align with the Figma template. This will ensure that the overall presentation looks more professional and visually appealing to viewers.

**Your design**

![Screenshot from 2023-04-28 12-31-01](https://user-images.githubusercontent.com/105117832/235215414-7d6cdfe0-a8e0-4389-8a11-984f8a6cabd8.png)

**Figma**

![Screenshot from 2023-04-28 12-30-50](https://user-images.githubusercontent.com/105117832/235215459-2550bdac-a16b-471d-9164-20942eaccd77.png)